fastlane/Fastfile

```diff
@@ -397,21 +397,17 @@ platform :ios do
     ensure_git_status_clean
 
     new_version = options[:version] || UI.input('Version number for the new hotfix?')
-    build_code_hotfix = build_code_hotfix(release_version: new_version)
 
     # Parse the provided version into an AppVersion object
     parsed_version = VERSION_FORMATTER.parse(new_version)
+    build_code_hotfix = BUILD_CODE_FORMATTER.build_code(version: parsed_version)
     previous_version = VERSION_FORMATTER.release_version(VERSION_CALCULATOR.previous_patch_version(version: parsed_version))
```

Comment:
Because `parsed_version` is used by both `build_code_hotfix` and `previous_version`, I decided to just call `BUILD_CODE_FORMATTER` directly in the `build_code_hotfix` assignment instead of having a separate method (removed below). I think this makes more sense as `build_code_hotfix` is only used in this one instance anyway.